src/MeetingsSDKAdapter.js

```diff
@@ -295,16 +306,25 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     const sdkMeeting = this.fetchMeeting(ID);
 
     try {
+      const isInSession = this.meetings[ID].remoteAudio !== null;
```

Comment:
You can more accurately determine if the meeting is joined by inspecting the `meeting.meetingFiniteStateMachine.state` property. The constant to check would be `MEETING_STATE_MACHINE.STATES.JOINED`

Reply:
is this publicly supported? I might have missed it in the docs.